content/news/buzz/2017-google-summer-of-code-final-submission-adam-boniecki/contents.lr

```diff
@@ -0,0 +1,54 @@
+title: 2017 Google Summer of Code - Final Submission
+---
+author: Adam Boniecki
+---
+body:
+
+With Google Summer of Code 2017 program nearing its end, it is time to summarize what I got done during the summer working on `Batavia </project/projects/bridges/batavia/>`__.
+
+Batavia is a part of Pybee's BeeWare collection of projects. I posted my proposal in `this GitHub thread <https://github.com/pybee/batavia/issues/483>`__ and kept it updated with my progress on a weekly basis.
+
+Note that by the end of GSoC, we have decided to diverge from the proposal and forgo implementation of `contextlib` in favor of support for Python 3.6 2-byte wide opcodes.
+
+Overall it was great learning experience and fun. Big thanks to my mentors Russel Keith-Magee and Katie McLaughlin, and the whole Pybee community.
+
+Pull Requests
+=============
+Lists and dicts
```

Comment:
General thoughts on the list below: Adding a short description for each link (and making them real links) would be super helpful. The description could even be the PR title. The goal here is to give readers a sense of what was accomplished in each category.